### PR TITLE
adding hindi

### PR DESCRIPTION
**Pull request by pat-schmitt**
Thanks @sandipjnu

### CODE REVIEW

**Comment by pat-schmitt** [comment]
Thanks @sandipjnu, could you please check your changes one more time using this link before we merge: https://mybinder.org/v2/gh/sandipjnu/world-glacier-explorer/sandipjnu-patch-1?urlpath=panel/app

File: international.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
-supported_languages = ['en', 'de', 'fr', 'es', 'it', 'nl', 'sv', 'ca', 'pt', 'lux', 'cn', 'fa', 'ur']
+supported_languages = ['en', 'de', 'fr', 'es', 'it', 'nl', 'sv', 'ca', 'pt', 'lux', 'cn', 'fa',
+                       'ur', 'hi']
```
Comment (fmaussion) [inline international.py:3]: put hindi after chinese (farsi and urdu are left to right languages)

**Review by fmaussion — COMMENTED** [review]
Thanks! Did you check that the languages fit into the app width?

**Comment by pat-schmitt** [comment]
oh sorry there is a bug in your changes in international.py, so the link is not working. Will fix this.

File: international.py (hunk 1 above)
Comment (pat-schmitt) [reply international.py:3]: Yes I have checked that all languages fits. And we can change the order

**Pull request closed without merging**